modules/apps/static/portal-osgi-web/portal-osgi-web-servlet-context-helper-impl/src/main/java/com/liferay/portal/osgi/web/servlet/context/helper/internal/ServletContextHelperFactoryImpl.java

```diff
@@ -35,10 +37,13 @@
 public class ServletContextHelperFactoryImpl
 	implements ServletContextHelperFactory {
 
-	@Activate
-	protected void activate(
-			BundleContext bundleContext, Map<String, Object> properties)
-		throws Exception {
+	@Override
+	public void ensureReady() {
+		if (_ready) {
```

Comment:
You don't need this condicion, because the check is already done in DCLSingleton